frappe/email/doctype/email_group_member/email_group_member.py

```diff
@@ -15,5 +16,9 @@ def after_insert(self):
 		email_group = frappe.get_doc('Email Group', self.email_group)
 		email_group.update_total_subscribers()
 
+	def validate(self):
+		if self.email:
+			validate_email_address(self.email, True)
```

Comment:
@prssanna Can't this be done by adding the option "Email" for the Docfield instead?